hudi-client/hudi-spark-client/pom.xml

```diff
@@ -31,6 +31,13 @@
   <packaging>jar</packaging>
 
   <dependencies>
+    <!-- Scala -->
+    <dependency>
+      <groupId>org.scala-lang</groupId>
+      <artifactId>scala-library</artifactId>
+      <version>${scala.version}</version>
+    </dependency>
+
```

Comment:
does it exist before？

Reply:
> does it exist before？
no, I moved some utils(`AvroConversionHelper`,`AvroConversionUtils`) from `hudi-spark`, they are implemented in scala.